test/api/branch-stream.test.js

```diff
@@ -16,32 +16,62 @@ test('branchStream', (t) => {
       pull.collect((err, branches) => {
         if (err) return cb(err)
 
-        t.equal(branches.length, 5, '5 branches')
-
-        t.equal(branches[0].length, 1, 'root mf alone')
-        t.equal(typeof branches[0][0][0], 'string', 'root mf alone')
+        t.equal(branches.length, 7, '7 branches')
+        const [
+          root,
+          rootV1,
+          rootV1Shard,
+          rootV1ShardMain,
```

Comment:
I looked at this thinking it was wrong for a long time.
I think a comment like "note rootChess etc were all added using the advanced API, skipping the v1/shard pattern"

Reply:
Done